pkg/garden/app/docket.hoon

```diff
@@ -348,11 +359,12 @@
   ++  new-chad  |=(c=chad (~(jab by charges) desk |=(charge +<(chad c))))
   ++  fetch-glob
     =/  =charge  (~(got by charges) desk)
+    ~&  charge+charge
     =/  tid=@t  (cat 3 'docket-' (scot %uv (sham (mix eny.bowl desk))))
     ?>  ?=(%glob -.href.docket.charge)
-    ?>  ?=(%http -.glob-location.href.docket.charge)
-    =*  url  url.glob-location.href.docket.charge
-    =/  =cage  spider-start+!>([~ `tid byk.bowl(r da+now.bowl) %glob !>(`url)])
+    =*  loc  glob-location.href.docket.charge
+    ~&  loc+loc
+    =/  =cage  spider-start+!>([~ `tid byk.bowl(r da+now.bowl) %glob !>(`[loc desk])])
```

Comment:
past 80 chars

Reply:
> The "{}" seems similarly redundant
> […](#)
> On Wed, Sep 1, 2021 at 15:06, Ted Blackman ***@***.***> wrote: ***@***.**** commented on this pull request. ------------------------------ In pkg/base-dev/lib/docket.hoon <[#5196 (comment)](https://github.com/urbit/urbit/pull/5196#discussion_r700606211)>: > @@ -91,6 +98,7 @@ ?+ -.clause "'{(trip +.clause)}'" %color (scow %ux color.clause) %site (spud path.clause) + %glob-ames ~& ship+ship.clause "{(scow %p ship.clause)}" stray printf -- should either remove or make look nice for production — You are receiving this because you are subscribed to this thread. Reply to this email directly, view it on GitHub <[#5196 (review)](https://github.com/urbit/urbit/pull/5196#pullrequestreview-744448017)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AAOFPBWRQIOWRHRELQJBAK3T72PVFANCNFSM5DFWSKIQ> . Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

A side effect of making a bunch of dumb changes to suss out an unrelated compile error. I do miss having error highlighting sometimes...